src/index.js

```diff
@@ -59,16 +87,23 @@ module.exports = function(session) {
         return callback(err);
       }
 
+      const createdAt = new Date();
+      const data = [
+        {
+          name: 'data',
+          value: sessJson,
+          excludeFromIndexes: true,
+        },
+        {
+          name: 'createdAt',
```

Comment:
Let's add keys only if someone has opted in to the feature, by setting a non-0 expiry? that way someone needs to opt in.

Reply:
was just chatting with @BenWhitehead, I actually think adding the date all the time is smarter, then it's easier to cleanup data.

Reply:
Sounds good to me, IIRC that was my reasoning as well.